macros/src/lib.rs

```diff
@@ -61,9 +56,24 @@ fn try_test(attr: TokenStream, input: ItemFn) -> syn::Result<Tokens> {
   let logging_init = expand_logging_init(&attribute_args);
   let tracing_init = expand_tracing_init(&attribute_args);
 
+  let (inner_test, generated_test) = if attr.is_empty() {
+    let test_attr: syn::Attribute = parse_quote! { #[::core::prelude::v1::test] };
+    let has_test = ignored_attrs.iter().any(|attr| *attr == test_attr);
+    let generated_test = if has_test {
+      quote! {}
+    } else {
+      quote! { #test_attr }
+    };
+    (quote! {}, generated_test)
+  } else {
+    let attr: Tokens = attr.into();
```

Comment:
Personal style, but let's use `From` instead, for the sake of consistency?
```suggestion
    let attr = Tokens::from(attr);
```